composer.json

```diff
@@ -5,6 +5,8 @@
 	"require": {
 	},
 	"require-dev": {
-		"codeception/codeception": "^2.4"
+		"codeception/codeception": "^2.4",
```

Comment:
We will probably just need PHPUnit here. @Solomonic ?

Reply:
Yes we can. 
But I feel codeception `example` (test data set) when using `specify` package is a bit more clean than using `@dataProvider` annotation from PHPUnit.
And also builtin DB assertion like `seeInDatabase()`, etc.

Do we have to setup thing a lot for make PHPUnit works with Selenium ?

Reply:
Codeception is fine in my book, I gave Lalita the choice to pick and she went with it.  It comes with some neat functions and syntax, we are in the scope of the package so we are good.

Reply:
for fullstack testing codeception is great otherwise it is overkill.
here phpunit should be well enough.
but again it really doesnt matter as tests are not production codes